RiotSwiftUI/Modules/Room/PollHistory/Service/PollHistoryServiceProtocol.swift

```diff
@@ -14,6 +14,17 @@
 // limitations under the License.
 //
 
+import Combine
+
 protocol PollHistoryServiceProtocol {
-    func fetchHistory() async throws -> [PollListData]
+    /// Returns a Publisher publishing the polls in the next batch.
+    /// Implementations should return the same publisher if `next()` is called again before the previous publisher completes.
+    func next() -> AnyPublisher<TimelinePollDetails, Error>
```

Comment:
`nextBatch` might be a clearer name for this, as `pollHistoryService.next()` is a little unclear as to what will happen.

Reply:
Done here!
https://github.com/vector-im/element-ios/pull/7293/commits/11f605ba154e344c52546735a51f7b077c707df0